parlai/core/build_data.py

```diff
@@ -155,17 +175,6 @@ def untar(path, fname, deleteTar=True):
         os.remove(fullpath)
 
 
-def cat(file1, file2, outfile, deleteFiles=True):
```

Comment:
I notice this function is used here: https://github.com/facebookresearch/ParlAI/blob/master/parlai/tasks/twitter/build.py#L95